Source/WebCore/rendering/style/RenderStyleInlines.h

```diff
@@ -435,6 +436,7 @@ inline GapLength RenderStyle::initialRowGap() { return { }; }
 constexpr RubyPosition RenderStyle::initialRubyPosition() { return RubyPosition::Before; }
 inline Length RenderStyle::initialScrollMargin() { return zeroLength(); }
 inline Length RenderStyle::initialScrollPadding() { return { }; }
+inline std::optional<class ScrollbarColor> RenderStyle::initialScrollbarColor() { return std::nullopt; }
```

Comment:
`optional<class ScrollbarColor>` is a unusual choice. Usually we do something like `optional< ScrollbarColor>` or such.

Reply:
I may be misunderstanding the issue but I repeatedly got errors about template arguments needing to be a type when I didn't specify class. Potentially due to scrollbar color being an enum value (the CSS keyword colour)

Reply:
`ScrollbarColor` is not an enum ; `std::optional<ScrollbarColor>` should work fine.

Reply:
ScrollbarColor is a value in the ThemeColor enum and that causes issues (fails to build) with referencing ScrollbarColor without the class keyword in front.

Reply:
Indeed, the ThemeColor should be an `enum class` to prevent this.
Is there a way to avoid the include of `ThemeColor` though ?

Reply:
Thanks for that. I'll rebase this and drop the class word, once that's merged

Reply:
Made that and the other requested changes 🙂